client/src/views/Homepage/index.js

```diff
@@ -0,0 +1,98 @@
+import React, { Component } from "react";
+
+import style from "./Homepage.module.css";
+
+import { Grid, Button, TextField } from "../../components/Generic";
+import { getWorkSpaceData } from "../../apis/getWorkSpace";
+
+class Homepage extends Component {
```

Comment:
you can switch to functional components and use hooks over there.

Reply:
ok will change to functional.